### DISCUSSION TITLE
DİJİTAL DEPREM GÜVENLİĞİ PLATFORMU

### DISCUSSION THREAD

**Discussion by zgnruya** [acikkaynak/deprem-yardim-projesi#102] [Ideas]
Merhaba, 

2020 yılında Güven İstanbul adında bir dijital deprem güvenliği platformu için çalışmaya başladık, sosyal girişim olarak kuluçka sürecinden vs geçtik ancak farklı sebeplerle bu platformu hayata geçiremedik. Bugün görüyorum ki düşünülen bütüncül sisteme hala çok ihtiyaç var ve deprem öncesi/sırası/ sonrası için de işlev gösterebilecek böyle bir platformu hayata geçirmek istiyorum. Ben teknik bir insan değilim (avukatım, kent bilimcisiyim, kamuda  kentsel dönüşüm ve konut politikaları uzmanı olarak stratejik plan ekibinde çalışıyorum.), dolayısıyla yazılım ve haritalama noktasında becerim yok. Önemli notlar: 

- Bina bazlı veriye ulaşmak ilk etapta çok zor. kamu kurumları paylaşmıyor. ancak belli raporlar farklı ölçeklerde risk bilgisi vermekte. bunlarla başlayıp kamuyu paylaşmaya zorlamak bir seçenek. 

- Çalışmanın revize edilmesi gerekiyor. Sunumu bir miktar güncelledim, fikir vermesi açısından paylaşıyorum. 

- Oldukça kapsamlı, fazlar halinde çalışılması gereken, mümkünse geniş bir toplulukla birlikte ve bu alanda çalışan başka aktörlerle koordineli ilerlemesi arzulanan bir iş olarak kurgulamıştık. Hala da böyle yürümesi gerekir diye düşünüyorum. 

Benzer fikirler paylaşılmıştı sanıyorum, belki çerçeveyi yeniden kurmak için bir araya gelinebilir. 

[Dijital deprem güvenliği platformu_V2pptx.pptx](https://github.com/acikkaynak/afet-org/files/10721838/Dijital.deprem.guvenligi.platformu_V2pptx.pptx)

**Comment by zenepc** [comment]
Merhaba, @zgnruya ile iletişim kurarak bu tarz "olası senaryolar için deprem güvenliği platformu" üzerine neler yapılabilir konuştuk. Burada daha önceden paylaşılmış benzer fikirler mevcut, linklerini bırakıyorum: 
[https://github.com/acikkaynak/afet-org/discussions/75](url) @ramazansubasi
[https://github.com/acikkaynak/afet-org/discussions/87](url) @alisaitteke

Olası senaryolar ile ilgili yapı bilgileri, sorgulamaları, deprem güvenliği analizlerinin haritalandığı bir platformun kurulması yönünde bir talep olduğunu düşünüyorum. Fikirlerin ve emek verecek insanların bir araya gelmesiyle bu yönde bir çalışma yapmak, @zgnruya'nın bahsettiği çalışmayı genişletmek için planlama yapılabilir diye düşünüyorum.

**Reply by zgnruya** [reply]
iletişim için ozguruya@gmail.com
